freelists.go

```diff
@@ -1,70 +0,0 @@
-// Copyright 2015 Google Inc. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package fuse
-
-import (
-	"unsafe"
-
-	"github.com/jacobsa/fuse/internal/buffer"
-)
-
-////////////////////////////////////////////////////////////////////////
-// buffer.InMessage
-////////////////////////////////////////////////////////////////////////
-
-// LOCKS_EXCLUDED(c.mu)
-func (c *Connection) getInMessage() *buffer.InMessage {
```

Comment:
The functions in this file were essentially just copied to `DefaultMessageProvider`